src/schedule/ll_schedule.c

```diff
@@ -288,46 +280,16 @@ int schedule_ll_task_init(struct task *task, uint16_t type, uint16_t priority,
 			  enum task_state (*run)(void *data), void *data,
 			  uint16_t core, uint32_t flags)
 {
-	struct ll_task_pdata *ll_pdata;
-	int ret = 0;
-
-	ret = schedule_task_init(task, type, priority, run, data, core, flags);
-	if (ret < 0)
-		return ret;
-
-	if (ll_sch_get_pdata(task))
-		return -EEXIST;
-
-	ll_pdata = rzalloc(RZONE_SYS_RUNTIME, SOF_MEM_CAPS_RAM,
-			   sizeof(*ll_pdata));
-
-	if (!ll_pdata) {
-		trace_ll_error("schedule_ll_task_init() error: alloc failed");
-		return -ENOMEM;
-	}
-
-	/* flush for slave core */
-	if (cpu_is_slave(task->core))
-		dcache_writeback_invalidate_region(ll_pdata,
-						   sizeof(*ll_pdata));
-
-	ll_sch_set_pdata(task, ll_pdata);
-
-	return 0;
+	return schedule_task_init(task, type, priority, run, data, core, flags);
```

Comment:
@tlauda its not clear the above change is directly because of adding period to struct task. If not, it should be a separate patch?